sdp/pyfftw_sdp.py

```diff
@@ -7,36 +7,64 @@ class SLIDING_DOT_PRODUCT:
     def __init__(self):
         self.m = 0
```

Comment:
Is `self.m` even needed? 

It looks like it's only being used internally in `__call__`